applications/tari_signaling_server/src/data.rs

```diff
@@ -46,6 +47,7 @@ impl Data {
     pub fn generate_jwt(&mut self, permissions: JrpcPermissions) -> Result<String> {
         let my_claims = Claims {
             id: self.id,
+            name: self.id.to_string(),
```

Comment:
If the `name` is the same as `id`, we can just use the `id`. Or do we plan to have some ui on the signaling server where we can see the names?